src/markdoc/layouts/Post.svelte

```diff
@@ -152,6 +184,38 @@
 										</ul> -->
                                     </div>
                                 {/if}
+
+                                <div class="share-post-section u-flex u-gap-16 u-margin-block-start-16 u-cross-center">
+                                    <span
+                                        class="u-font-size-16 u-padding-inline-end-8"
+                                        style:font-weight="500">
+                                        SHARE
+                                    </span>
+
+                                    <ul class="u-flex u-gap-8">
+                                        {#each socialSharingOptions as sharingOption}
+                                            <li class="share-list-item">
+                                                <Tooltip placement="bottom" disableHoverableContent={true}>
+                                                    <button
+                                                        class="web-icon-button"
+                                                        aria-label={sharingOption.label}
+                                                        on:click="{() => handleShareClick(sharingOption)}"
```

Comment:
lets use an anchor here, buttons are not that accessible 👍🏻 

you can use the same function, change it a bit and return the `href`

Reply:
What about `Copy to clipboard`?

Reply:
Added separate tags for links and copy button.